src/index.js

```diff
@@ -417,7 +426,7 @@ function swap (arr, i, n) {
 }
 
 
-function changeOrientation(contour) {
+function changeOrientation(contour) {  // eslint-disable-line no-unused-vars
```

Comment:
Yeah that was just a hangover from the past code, it's worth highlighting to @w8r that it's no longer used, he might have had some ideas as to why it was there originally....